### DISCUSSION TITLE
Claim (no longer) available

### DISCUSSION THREAD

**Discussion by type2diabetes** [RakambdaOrg/ChannelPointsMiner#263] [General]
Let me just start by saying this has most likely nothing to do with the miner, just asking here to see if it's just me having the issue or others too, and maybe someone knows what's going on.

So, about 12-13 hours ago everything was working fine, and I got the +50 claims every 15 minutes like usual;

![atfirstiwaslike](https://user-images.githubusercontent.com/86139709/187389570-c5e4efe6-13e8-47c9-aa34-020b72e06945.png)

but this morning I wake up and I see it's no longer doing that, it seems to only get the claim once when a streamer comes online, then never again.

![butthen](https://user-images.githubusercontent.com/86139709/187389566-32fe8f3b-722e-41f0-996d-f2498ae7f9d3.png)

The miner have been running since a few days now, it just randomly started happening today.
Anyone else noticed this?

**Comment by Rakambda** [comment]
I just looked and have the same issue. I have however 1 +50 that appeared randomly around 11:58 CEST.

Maybe twitch api isn't going too well? If it continues over time I'll look deeper into it.

**Reply by Rakambda** [reply]
I'm not able to do so right now but a thing to maybe look into is see if it is also broken on twitch website.

**Reply by type2diabetes** [reply]
Seems to be broken on the website too, atleast on my end.

Checked Twitter and seems others have been complaining about the same thing before and around the same timeframe.
Lets see if it's fixed after their maintenance tonight _(or if they just add even more ads)_.. :)
![image](https://user-images.githubusercontent.com/86139709/187473244-27adc83c-fa1a-4c38-bac6-b44978f498f5.png)

**Comment by type2diabetes** [comment]
Alright, it started working again an hour ago.
I saw on Twitter yesterday that they were aware of the problem, so seems they finally got around to fixing it. :)
